tests/src/test/scala/whisk/core/controller/test/MetaApiTests.scala

```diff
@@ -942,19 +947,33 @@ trait MetaApiTests extends ControllerTestCommon with BeforeAndAfterEach with Whi
                 foreach { path =>
                     actionResult = Some(JsObject("text" -> "Something".toJson))
                     Post(s"$testRoutePath/$path", "This is the body") ~> addHeader("x-ow-raw-http", "false") ~> addHeader("Content-type", MediaTypes.`text/html`.value) ~> sealRoute(routes(creds)) ~> check {
-                        status should be(UnsupportedMediaType)
+                        status should be(BadRequest)
+                        confirmErrorWithTid(responseAs[JsObject], Some(Messages.contentTypeNotSupported))
                     }
                 }
         }
 
+        it should s"invoke a web action using the x-ow-raw-http header (auth? ${creds.isDefined})" in {
+            implicit val tid = transid()
+
+            Seq(s"$systemId/proxy/export_c.text").
+                    foreach { path =>
+                        actionResult = Some(JsObject("text" -> "Something".toJson))
+                        Post(s"$testRoutePath/$path", "This is the body") ~>  addHeader("x-ow-raw-http", "TruE") ~> addHeader("Content-type", MediaTypes.`text/html`.value) ~> sealRoute(routes(creds)) ~> check {
+                            status should be(OK)
```

Comment:
something like this: `action.fields(”content”).asJsObject.fields(”__ow_meta_body”) shouldBe Some(JsString(”Something”))`